python/rmm/rmm/_lib/memory_resource.pyx

```diff
@@ -1188,3 +1188,10 @@ def get_log_filenames():
         else None
         for i, each_mr in _per_device_mrs.items()
     }
+
+
+def get_free_device_memory():
+    """
+    Return the number of bytes of available device memory.
+    """
+    return percent_of_free_device_memory(100)
```

Comment:
This seems like it should be calling `rmm::available_device_memory` and returning the "free" amount, without any percent-based functions.

https://github.com/rapidsai/rmm/blob/9c5d48680add334268704d99095137534c276874/include/rmm/cuda_device.hpp#L111

Reply:
Thanks! Using this function.